postprocessing-analysis/postprocessing-analysis.py

```diff
@@ -1,17 +1,29 @@
 from kafka import KafkaConsumer
 from pymongo import MongoClient
-from json import loads
+
+from kafka import KafkaProducer
+import json
+import copy
 
 consumer = KafkaConsumer(
     'postprocessing-analysis',
      bootstrap_servers=['localhost:9092'],
-     value_deserializer=lambda x: loads(x.decode('utf-8')))
+     value_deserializer=lambda x: json.loads(x.decode('utf-8')))
+
+producer = KafkaProducer(bootstrap_servers=['localhost:9092'],
+                         value_serializer=lambda x: 
+                         json.dumps(x, default = str).encode('utf-8'))
 
 client = MongoClient('localhost:27020')
 db = client.postprocess
 
 while True:
     message = consumer.poll()
     if bool(message):
-        postprocess.data.insert_one(message)
-        print('{} added to {}'.format(message, db))
+        _,v = message.popitem()
+        weather = v[0].value
+        print(weather)
+        mongosend = copy.deepcopy(weather)
+        result = db.weather.insert_one(mongosend)
+
```

Comment:
for now post processing is showing entire weather details or same as model execution?

Reply:
Yes this will be improved in future PRs